frontend/src/scenes/insights/views/Funnels/FunnelCorrelation.tsx

```diff
@@ -65,6 +65,8 @@ export const FunnelCorrelation = (): JSX.Element | null => {
 
                     <FunnelCorrelationTable />
 
+                    <FunnelPropertyCorrelationTable />
```

Comment:
curious why this change?

Reply:
(this makes feedback form go below the fold and probs invisible even when it becomes visible at the bottom)

Reply:
are we still looking at any of the feedback from it? I don't love the way the emojis look so I'd argue for removing it entirely if not

Reply:
I usually reference it whenever someone comes to me to ask 'Are correlation analysis useful? Still used?' - but happy to stick with existing data and get rid of this form too, specially since it's out of our team's purview now

Reply:
https://app.posthog.com/dashboard/19702

Reply:
So this was an unrelated change for sure - I just found it weird having it in the middle of both tables.

Happy to revert though